lib/Home.dart

```diff
@@ -116,23 +121,50 @@ class _HomeState extends State<Home> {
     );
   }
 
+  Widget _profilePage(BuildContext context) {
+    return Column (
+      children: <Widget>[
+        Profile(),
+        Padding (
+          padding: EdgeInsets.only(top: 6),
+          child: AccountInfo()
+        )
+      ],
+    );
+  }
+  Widget getPage(BuildContext context, int index) {
+    switch (index) {
+      case (RIDES):
+        if (rides.length == 0) return _noRidesLeftPage(context);
+        else return _ridesPage(context);
+        break;
```

Comment:
Are the "break" statements within each case necessary? Wouldn't they never be reached because of the return statements?

Reply:
Oh right, didn't think of that--was just autopilotting on what I remembered of switch statement syntax 😅 I'll remove those!